news/2024/2024-05-20-the-perennial-returns.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: "The Perennial 2024 Returns"
```

Comment:
if you're talking about the series, this might make more sense

```suggestion
title: "The Perennial Returns"
```

Reply:
ye dont think having 2024 there makes sense